### ISSUE TITLE
Can't add any library - plugins issue ?

### ISSUE THREAD

**Issue by antoninblanc** [comio/plex-overlay#62]
I can't add any library after update to latest version. "Next" button is grey out. 

![plex-add-library](https://github.com/comio/plex-overlay/assets/1034227/c652d5b3-e124-4af7-9e43-437610b05006)

Here is `Plex Media Server.log` It seams there are some issues with plugins, but not sure if it's normal.

> ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
> ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.agents.themoviedb], we're killing it.
> ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.agents.thetvdb], we're killing it.
...

```
Apr 27, 2024 16:46:24.053 [140535296322360] INFO - Plex Media Server v1.40.2.8395-c67dce28e - Gentoo PC x86_64 - build: linux-x86_64 debian - GMT 02:00
Apr 27, 2024 16:46:24.054 [140535296322360] INFO - Linux version: default/linux/amd64/23.0/no-multilib/systemd, language: en-US
Apr 27, 2024 16:46:24.054 [140535296322360] INFO - Processor: 12-core        Intel(R) Core(TM) i7-3930K CPU @ 3.20GHz
Apr 27, 2024 16:46:24.054 [140535296322360] INFO - Compiler is - Clang 11.0.1 (https://plex.tv 9b997da8e5b47bdb4a9425b3a3b290be393b4b1f)
Apr 27, 2024 16:46:24.054 [140535296322360] INFO - /usr/lib/plexmediaserver/Plex Media Server
Apr 27, 2024 16:46:24.149 [140535298886288] INFO - [CERT/OCSP] Successfully retrieved response from cache.
Apr 27, 2024 16:46:24.251 [140535298886288] INFO - Running migrations. (EPG 0)
Apr 27, 2024 16:46:24.307 [140535298886288] INFO - Running migrations. (EPG 0)
Apr 27, 2024 16:46:26.524 [140535225350968] ERROR - BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:46:26.524 [140535225350968] ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
Apr 27, 2024 16:46:26.524 [140535225350968] ERROR - Error received reading configuration for com.plexapp.system
Apr 27, 2024 16:46:26.524 [140535294118712] WARN - JobManager: Could not find job for handle 151238
Apr 27, 2024 16:46:27.087 [140535289310008] WARN - [EventSourceClient/pubsub/139.162.215.242:443] MyPlex: attempted a reachability check but we're not yet mapped.
Apr 27, 2024 16:46:28.663 [140535225350968] ERROR - BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:46:28.663 [140535225350968] ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
Apr 27, 2024 16:46:28.663 [140535225350968] ERROR - Error received reading configuration for com.plexapp.system
Apr 27, 2024 16:46:28.663 [140535294118712] WARN - JobManager: Could not find job for handle 151259
Apr 27, 2024 16:46:30.406 [140535175158584] INFO - [Req#25] AutoUpdate: no updates available
Apr 27, 2024 16:46:30.809 [140535183727416] ERROR - BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:46:30.809 [140535183727416] ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.agents.themoviedb], we're killing it.
Apr 27, 2024 16:46:30.809 [140535183727416] ERROR - Error received reading configuration for com.plexapp.agents.themoviedb
Apr 27, 2024 16:46:30.809 [140535294118712] WARN - JobManager: Could not find job for handle 151296
Apr 27, 2024 16:46:30.837 [140535201569592] ERROR - BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:46:30.837 [140535201569592] ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.agents.plexthememusic], we're killing it.
Apr 27, 2024 16:46:30.837 [140535201569592] ERROR - Error received reading configuration for com.plexapp.agents.plexthememusic
Apr 27, 2024 16:46:30.837 [140535294118712] WARN - JobManager: Could not find job for handle 151299
Apr 27, 2024 16:46:30.844 [140535185836856] ERROR - BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:46:30.844 [140535185836856] ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.agents.thetvdb], we're killing it.
Apr 27, 2024 16:46:30.844 [140535185836856] ERROR - Error received reading configuration for com.plexapp.agents.thetvdb
Apr 27, 2024 16:46:30.844 [140535294118712] WARN - JobManager: Could not find job for handle 151297
Apr 27, 2024 16:46:32.935 [140535231757112] ERROR - BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:46:32.935 [140535231757112] ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
Apr 27, 2024 16:46:32.935 [140535231757112] ERROR - Error received reading configuration for com.plexapp.system
Apr 27, 2024 16:46:32.935 [140535294118712] WARN - JobManager: Could not find job for handle 151312
Apr 27, 2024 16:46:35.082 [140535183727416] ERROR - BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:46:35.082 [140535183727416] ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.agents.imdb], we're killing it.
Apr 27, 2024 16:46:35.082 [140535183727416] ERROR - Error received reading configuration for com.plexapp.agents.imdb
Apr 27, 2024 16:46:35.082 [140535294118712] WARN - JobManager: Could not find job for handle 151334
Apr 27, 2024 16:46:35.093 [140535227530040] ERROR - BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:46:35.093 [140535227530040] ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
Apr 27, 2024 16:46:35.093 [140535294118712] WARN - JobManager: Could not find job for handle 151336
Apr 27, 2024 16:46:35.093 [140535227530040] ERROR - Error received reading configuration for com.plexapp.system
Apr 27, 2024 16:46:37.209 [140535214803768] ERROR - Unknown metadata type: 
Apr 27, 2024 16:46:37.226 [140535242312504] ERROR - BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:46:37.226 [140535242312504] ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
Apr 27, 2024 16:46:37.226 [140535242312504] ERROR - Error received reading configuration for com.plexapp.system
Apr 27, 2024 16:46:37.226 [140535294118712] WARN - JobManager: Could not find job for handle 151354
Apr 27, 2024 16:46:37.277 [140535227530040] ERROR - Unknown metadata type: 
....
Apr 27, 2024 16:46:56.541 [140535219022648] ERROR - Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
Apr 27, 2024 16:46:56.542 [140535219022648] ERROR - Error received reading configuration for com.plexapp.system
Apr 27, 2024 16:46:56.542 [140535294118712] WARN - JobManager: Could not find job for handle 151516
Apr 27, 2024 16:46:56.557 [140535235975992] ERROR - [Req#4b] BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:46:56.557 [140535235975992] ERROR - [Req#4b] Timeout or error reading status line from plug-in pipe [com.plexapp.agents.imdb], we're killing it.
Apr 27, 2024 16:46:56.557 [140535235975992] ERROR - [Req#4b] Error received reading configuration for com.plexapp.agents.imdb
Apr 27, 2024 16:46:56.557 [140535294118712] WARN - JobManager: Could not find job for handle 151519
Apr 27, 2024 16:46:56.557 [140535235975992] ERROR - downloadContainer: expected MediaContainer element, found html
Apr 27, 2024 16:46:56.557 [140535235975992] ERROR - IVA: failed to get IMDB prefs
Apr 27, 2024 16:49:01.117 [140535208442680] ERROR - [Req#71] BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:49:01.117 [140535208442680] ERROR - [Req#71] Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
Apr 27, 2024 16:49:01.117 [140535208442680] ERROR - [Req#71] Error received reading configuration for com.plexapp.system
Apr 27, 2024 16:49:01.117 [140535294118712] WARN - JobManager: Could not find job for handle 151644
Apr 27, 2024 16:49:05.893 [140535208442680] ERROR - [Req#73] BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:49:05.893 [140535208442680] ERROR - [Req#73] Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
Apr 27, 2024 16:49:05.893 [140535208442680] ERROR - [Req#73] Error received reading configuration for com.plexapp.system
Apr 27, 2024 16:49:05.893 [140535294118712] WARN - JobManager: Could not find job for handle 151683
Apr 27, 2024 16:49:12.898 [140535208442680] ERROR - [Req#74] BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:49:12.898 [140535208442680] ERROR - [Req#74] Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
Apr 27, 2024 16:49:12.898 [140535208442680] ERROR - [Req#74] Error received reading configuration for com.plexapp.system
Apr 27, 2024 16:49:12.898 [140535294118712] WARN - JobManager: Could not find job for handle 151695
Apr 27, 2024 16:49:34.151 [140535208442680] ERROR - [Req#75] BufferingLineReader: failed to read line (error: -1)
Apr 27, 2024 16:49:34.151 [140535208442680] ERROR - [Req#75] Timeout or error reading status line from plug-in pipe [com.plexapp.system], we're killing it.
Apr 27, 2024 16:49:34.151 [140535208442680] ERROR - [Req#75] Error received reading configuration for com.plexapp.system
Apr 27, 2024 16:49:34.151 [140535294118712] WARN - JobManager: Could not find job for handle 151719

```
I made a backup of my `/var/lib/plexmediaserver/` folder and restarted plex with an empty one, but same issue I can't add any library. I tried running `strace` but I couldn't find where those errors reported above come from, I guess plex spawn other process? And their app packaging isn't the most straightforward, to put it kindly....

Can someone confirm latest ebuild is working ? If so I'll open a ticket on Plex support.

Thanks.

**Comment by antoninblanc** [comment]
Oh, seams it can't find any plugins. 

![plex-plugins](https://github.com/comio/plex-overlay/assets/1034227/82707166-79cd-494f-8797-36b0a5d22604)

**Comment by comio** [comment]
I'm sorry but this is just a repacking and I cannot support any software bug. Try asking on  https://forums.plex.tv

**Comment by antoninblanc** [comment]
@comio Thanks. I know!  I'm just asking if someone can confirm plex working fine, to make sure it doesn't come from Gentoo or this ebuild. As I've mentionned `[if plex is working fine] I'll open a ticket on Plex support.`  Thanks again.

**Comment by comio** [comment]
My setup just working (v1.40.2.8395):

![immagine](https://github.com/comio/plex-overlay/assets/277587/3c53f19b-40ab-4746-8547-8e508173f142)

Do you have any dns or network issue?

**Comment by antoninblanc** [comment]
Ok.. thanks!!  Network is fine.

I've been able to remove the crash above with monkey patching uuid.py in their python2.7.zip Now plugins are able to start, sadly it doesn't fix my problem. I created an issue on their forum but i would be surprise to find help from plex team. It looks like an issue with their ugly python2... their software packaging is horrible, I might just move to Docker.

 https://forums.plex.tv/t/plugins-crash-plex-script-host-segfault-at-2380-ip-0000000000002380-sp/876309
